.travis.yml

```diff
@@ -4,36 +4,28 @@ sudo: false
 
 matrix:
   include:
-    - os: linux
-      python: 2.7
-    - os: linux
-      python: 3.4
     - os: linux
       python: 3.5
     - os: linux
-      python: pypy
-    - os: osx
-      language: generic
-      env:
-      - PYTHON_VERSION=2.7.10
-      - PYENV_ROOT=~/.pyenv
-      - PATH=$PYENV_ROOT/shims:$PYENV_ROOT/bin:$PATH
+      python: 3.6
+    - os: linux
+      python: pypy3
     - os: osx
       language: generic
       env:
-      - PYTHON_VERSION=3.4.3
+      - PYTHON_VERSION=3.5.6
       - PYENV_ROOT=~/.pyenv
       - PATH=$PYENV_ROOT/shims:$PYENV_ROOT/bin:$PATH
     - os: osx
       language: generic
       env:
-      - PYTHON_VERSION=3.5.0
+      - PYTHON_VERSION=3.6.8
```

Comment:
Maybe we could also add python 3.7

Reply:
TravisCI does not provide 3.7 :(

Reply:
You can use `dist: xenial` to get 3.7 support on travis, see https://github.com/scottwernervt/favicon/blob/master/.travis.yml#L15